pkg/morph/event/notary_preparator_test.go

```diff
@@ -24,8 +24,8 @@ var (
 	alphaKeys      keys.PublicKeys
 	wrongAlphaKeys keys.PublicKeys
 
-	dummyInvocationScript      = append([]byte{byte(opcode.PUSHDATA1), 64}, make([]byte, 64)...)
-	wrongDummyInvocationScript = append([]byte{byte(opcode.PUSHDATA1), 64, 1}, make([]byte, 63)...)
+	dummyAlphabetInvocationScript = []byte{} // expected to be empty, as requester can't fill it in.
```

Comment:
We can use both new and old one in tests.

Reply:
Fixed.